receipt_dynamo/analyze_mixin_consolidation.py

```
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
+"""
+Script to analyze how data access classes can be migrated to consolidated mixins.
+"""
+
+import os
+import re
+from collections import Counter
+from typing import Dict, List, Optional, Tuple
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)